google-cloud-spanner/src/test/java/com/google/cloud/spanner/ITSessionPoolIntegrationTest.java

```diff
@@ -97,28 +98,28 @@ public ScheduledExecutorService get() {
 
   @Test
   public void sessionCreation() {
-    try (Session session = pool.getReadSession()) {
-      assertThat(session).isNotNull();
+    try (PooledSessionFuture session = pool.getReadSession()) {
+      assertThat(session.get()).isNotNull();
```

Comment:
WDYT about adding timeouts to the `get` calls for the futures (in order not to block indefinitely)?

Reply:
I would prefer not do that, as the client library also calls `PooledSessionFuture.get()` without a timeout, and I would prefer to keep these consistent with the behavior of the client library itself. Instead, the test case itself will eventually timeout if the calls should block indefinitely.

Reply:
Cool, thanks for the explanation!